webforj-plugins/webforj-install-bbj-plugin/src/main/java/com/webforj/installer/WebforjInstaller.java

```diff
@@ -32,24 +28,25 @@
 import java.util.stream.Stream;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipFile;
-import javax.xml.parsers.ParserConfigurationException;
+import org.apache.commons.io.FilenameUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.maven.shared.invoker.DefaultInvocationRequest;
 import org.apache.maven.shared.invoker.DefaultInvoker;
 import org.apache.maven.shared.invoker.InvocationRequest;
 import org.apache.maven.shared.invoker.Invoker;
 import org.apache.maven.shared.invoker.MavenInvocationException;
-import org.xml.sax.SAXException;
+import org.apache.maven.shared.utils.StringUtils;
+
 
 /**
  * perform the installation of a Webforj app based on its JAR.
  */
+@SuppressWarnings({"unused"})
```

Comment:
Why? Remove please

Reply:
This is a class used externally only.  Sonar adds a complaint if I don't do this.

Reply:
Fix the complains , dont suppress warnings for all class

Reply:
I'll remove it.  This is a stupid sonar rule for a java library like this.
No other location in the project calls WebforjInstaller becuase it is only for the BBj plugins container to call.
Sonar only gives you so many errors before it refuses to pass.  This resolves the sonar issue without damaging or endangering the code.

Reply:
Warning:(45, 20) Class 'WebforjInstaller' is never used
Warning:(184, 8) Method 'deleteDirectory(java.io.File)' is never used
Warning:(206, 17) Method 'install(java.lang.String, java.lang.String, java.lang.String, java.lang.String)' is never used

Ask Stephan why he chose to design this class in this manner.  Is there a requrirement for the deleteDirectory method?  Why are there two variants of the install method, when only one is used by the BBj container.